File: Random/2.Longest Common Prefix.py
def solve(strs):
    n=len(strs)
    to_loop=100000
    for i in strs:
        to_loop=min(to_loop,len(i))
    print(to_loop)
    s=0
    for i in range(to_loop):
        s=strs[0][:i]
        for j in strs:
            if j[:i]==s:
                pass
            else:
                return s[:i-1]
    return s
print(solve(["flow","fer","flower"]))


        

File: Random/2.Longest Common Prefix.js
let solve1=function(strs){
    let to_loop=10000;
    n=strs.length;
    let s=""
    for(let i=0;i<n;i++){
        to_loop=Math.min(to_loop,strs[i].length);
    }
    for(let i=0;i<=to_loop;i++){
        s=strs[0].substring(0,i);
        strs.forEach(function(element){
            if(s==element.substring(0,i)){
                b=0
            }
            else{
                return s.substring(0,i-1)
            }
        })
    }
    return s
}
console.log(solve1(["flower","flow"]));